modules/swagger-codegen/src/main/resources/go-server/logger.mustache

```diff
@@ -0,0 +1,23 @@
+package {{packageName}}
+
+import (
+    "log"
+    "net/http"
+    "time"
+)
+
+func Logger(inner http.Handler, name string) http.Handler {
+    return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+        start := time.Now()
+
+        inner.ServeHTTP(w, r)
+
+        log.Printf(
+            "%s\t%s\t%s\t%s",
+            r.Method,
+            r.RequestURI,
+            name,
+            time.Since(start),
+        )
```

Comment:
Is this meant to be an example that you'd expect users to change completely, or do you expect users to "embrace and extend" from this foundation?